debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
This is the native way of building and packaging on a debian based machine. Alien is only useful if you already have an rpm package that you want to install on a debian machine. This is useful for somebody who does not want to first create an rpm package (on a machine supporting rpm) and then convert it to debian using alien. Like he/she wants to download source on their Ubuntu machine and make a debian package to distribute to his colleague, say.